indispensable/src/jvmMain/kotlin/at/asitplus/signum/indispensable/JcaExtensions.kt

```diff
@@ -17,14 +17,17 @@ import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo
 import org.bouncycastle.jce.ECNamedCurveTable
```

Comment:
please remove all whitespace-only changes from this file to make it ready for review